src/components/Databoard-review.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from "react";
+// import getData from "../getData";
+import ResultSegment from "./ResultSegment";
+
+export default function Databoard({ timekey }) {
+  // set state variables to store data to be displayed
+  const [dataLoaded, setDataLoaded] = useState(false);
+
+  const [homePossessionData, setHomePossessionData] = useState([]);
+  const [awayPossessionData, setAwayPossessionData] = useState([]);
+
+  const [homeShotData, setHomeShotData] = useState([]);
+  const [awayShotData, setAwayShotData] = useState([]);
+
+  const [homeShotOnTargetData, setHomeShotOnTargetData] = useState([]);
+  const [awayShotOnTargetData, setAwayShotOnTargetData] = useState([]);
+
+  const [homeCornersData, setHomeCornersData] = useState([]);
+  const [awayCornersData, setAwayCornersData] = useState([]);
+
+  useEffect(() => {
+    console.log("Fetching data");
+    fetch(process.env.REACT_APP_API_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        let matchData = data.match;
+        let liveData = matchData.liveData;
+        let lineupData = liveData.lineups;
+        let homeData = lineupData.home.stats;
+        let awayData = lineupData.away.stats;
```

Comment:
it looks like you're only using the awayData and homeData - I would remove all the variables not in use. Also worth making them `const`